src/components/App.tsx

```diff
@@ -2046,6 +2050,7 @@ class App extends React.Component<any, AppState> {
         this.setState({
           draggingElement: element,
           editingElement: element,
+          multiElement: element,
```

Comment:
This is probably causing the issue.

Reply:
Could you please explain a little bit what that property does? (At some point I was just copy pasting code from other places and I probably messed up somewhere :P )

Reply:
To be honest, I do not know. 😓 I just assumed this is only the place to cause such an issue. Probably, it's setting the `multiElement` even when it's only two or less elements. Could you try removing this line and see if the behavior changes? And maybe only add it when it's more than two points. Again, I'm just guessing.

Reply:
That's not it, although it was causing issues for arrow/line elements, and even some weird jumping for `draw` element -- so I removed it. Don't know what was the purpose of adding it, but if it's needed for `draw` elements, we'll need to adapt it since it's very much tied to `arrow`/`line` elements.